modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -276,6 +295,16 @@ private String _getAPIURL(
 		return _interpolateURL(sb.toString(), httpServletRequest);
 	}
 
+	private JSONObject _getFDSSortJSONObject(ObjectEntry fdsSort) {
```

Comment:
For the name here. I think we would probably want `_getSortsJSONObject` as this would be consistent with your other methods like `getSortsJSONArray`

Reply:
Thanks to notice it, @bryceosterhaus , just changed